go/src/socialapi/workers/oauth/api/api.go

```diff
@@ -1,100 +0,0 @@
-package main
```

Comment:
I think best way is to add a build tag, so the source code is still there but it will not pick up by any tools:

```go
// +build wip

package main
```